plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/SpanAddingClassVisitorFactory.kt

```diff
@@ -79,6 +80,9 @@ abstract class SpanAddingClassVisitorFactory :
              * from the sentry-android SDK.
              */
             val instrumentables = listOfNotNull(
+                ComposeNavigation().takeIf {
```

Comment:
Let's move this it into the `ChainedInsturmentable` because now the instrumentation will stop as soon as it reaches the first Insturmentable which return `true` in `isInstrumentable`